### ISSUE TITLE
Закрытие приложения полностью

### ISSUE THREAD

**Issue by anelfer** [grishka/Houseclub#66]
Хотелось бы после закрытия приложения из диспетчера задач, что бы кикало из комнаты и прила закрывалась полностью

**Comment by combrs** [comment]
да, меня тоже немного удивило, что голос из телефона продолжает идти, даже если выйти из приложения (не просто открыть другое), даже если попытаться его выгрузить из списка приложений... прекращается только если явно перед этим нажать Leave room
Нужно какое-то средство, например при нажатии "назад" в некоторых приложениях пишут "нажмите назад еще раз, если хотите закрыть приложение", и полностью закрываются.
